src/app/helpers.js

```diff
@@ -300,7 +300,7 @@ function getCurrentProjectId(projectIds) {
   const currentProjectId = window.location.pathname.match(/project\/([0-9]+)/);
   let projectId = currentProjectId ? parseInt(currentProjectId[1], 10) : null;
   if (projectId && projectIds) {
-    if (projectIds.indexOf(projectId) === -1) {
+    if (projectIds !== projectId) {
```

Comment:
Looks like the getCurrentProjectId and getCurrentProject helpers don't make sense anymore and can be removed.

Reply:
Removed both helpers